### PR TITLE
[FIX] Broken links

### PR DESCRIPTION
**Pull request by willeastcott**
Use the new dead link finder to repair all broken hyperlinks.

Also:
* Remove polyfills for `String.prototype.startsWith` and `String.prototype.endsWith` which have been supported since Node 4.0.0.
* Update deprecated (since Node 10.0.0) `new Buffer` calls to `Buffer.from`.
* Remove Kongregate page since they are no longer accepting new submissions from developers.

### CODE REVIEW

File: content/en/user-manual/publishing/web/index.md [hunk 1]
```diff
@@ -8,12 +8,10 @@ PlayCanvas games are actually web pages and as such, are most at home in the bro
 
 * [PlayCanvas Hosting][1]
 * [Self-hosting][2]
-* [Chrome Web Store][3]
-* [Kongregate][4]
-* [Facebook][5]
+* [Kongregate][3]
```
Comment (yaustar) [inline content/en/user-manual/publishing/web/index.md:11]: I don't think Kongregate are accepting new games from developers anymore

File: package-lock.json [hunk 2]
```diff
@@ -1007,11 +1007,6 @@
       "resolved": "https://registry.npmjs.org/is/-/is-3.3.0.tgz",
       "integrity": "sha512-nW24QBoPcFGGHJGUwnfpI7Yc5CdqWNdsyHQszVE/z2pKHXzh7FZ5GWhJqSyaQ9wMkQnsTx+kAI8bHlCX4tKdbg=="
     },
-    "is-absolute-url": {
```
Comment (yaustar) [inline package-lock.json:1010]: Slightly concerned that we are removing packages here

**Review by yaustar — APPROVED** [review]
Thanks! I've been meaning to do this all week!

File: package-lock.json (hunk 2 above)
Comment (willeastcott) [reply package-lock.json:1010]: This is a fresh `npm install` on macOS Big Sir and everything worked fine.

File: content/en/user-manual/publishing/web/index.md (hunk 1 above)
Comment (willeastcott) [reply content/en/user-manual/publishing/web/index.md]: I've removed the Kongregate page.

File: package-lock.json (hunk 2 above)
Comment (willeastcott) [reply package-lock.json:1010]: I'd suggest we just go with this and if there's a problem, it's very straightforward to back out.

Comment (yaustar) [reply package-lock.json:1010]: Sounds good

**Pull request merged**